mungegithub/Makefile

```diff
@@ -69,6 +69,12 @@ secret:
 clean:
 	rm -f mungegithub $(APP)/local.deployment.yaml $(APP)/local.secret.yaml
 
+# pull down current public queue state, and run UI based off that data
+ui-stub:
+	@/bin/bash -c "wget -q -r -nH -P ./submit-queue/www http://submit-queue.k8s.io/{prs,github-e2e-queue,history,sq-stats,stats,users,health,google-internal-ci,priority-info,merge-info,health}; \
```

Comment:
It's twice as important as the others ;) ;)

On Wed, Jun 15, 2016 at 10:56 AM, Ryan Hitchman <notifications@github.com>
wrote:

> In mungegithub/Makefile
> <https://github.com/kubernetes/contrib/pull/1209#discussion_r67211924>:
>
> > @@ -69,6 +69,12 @@ secret:
> >  clean:
> >  	rm -f mungegithub $(APP)/local.deployment.yaml $(APP)/local.secret.yaml
> >
> > +# pull down current public queue state, and run UI based off that data
> > +ui-stub:
> > +	@/bin/bash -c "wget -q -r -nH -P ./submit-queue/www http://submit-queue.k8s.io/{prs,github-e2e-queue,history,sq-stats,stats,users,health,google-internal-ci,priority-info,merge-info,health}; \
>
> this has health twice
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1209/files/4c965c6e8605a412fd0df245f2bdf1ac59eba6ea#r67211924>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAnglvdVBFD8PBlPKKwHWNOQQOFa7FsFks5qMDyxgaJpZM4I13qN>
> .
>